lib/modulesync/cli.rb

```diff
@@ -169,6 +169,32 @@ def execute(*command_args)
         ModuleSync.execute(config)
       end
 
+      desc 'reset', 'Reset all repositories'
```

Comment:
What does reset mean? Looking at the code I can figure it out but I think you can be a bit more verbose.

Reply:
Done: I explained it in long description